.github/assets/js/main.js

```
@@ -0,0 +1,332 @@
+                const tempDiv = document.createElement('div');
+                tempDiv.innerHTML = text;
+                
+                // Find all paper sections
+                const paperSections = tempDiv.querySelectorAll('h3');
+                const featuredSections = Array.from(paperSections).filter(section => {
+                    // Find the next tags element
+                    let nextEl = section.nextElementSibling;
+                    while (nextEl && !nextEl.matches('tags')) {
+                        nextEl = nextEl.nextElementSibling;
+                    }
+                    return nextEl && nextEl.textContent.includes('Featured');
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)